src/ItemEdit.js

```diff
@@ -139,9 +139,17 @@ const ItemEdit = ({ index, roadmap, onChange, onDone }) => {
 
   const handleSuggestionSelect = (index, event) => {
     const values = [...dateFields];
-    if (event.target.name && event.target.name.includes('Stage')) {
+    if (
+      event.target.innerText &&
+      roadmap.labels.map((label) => label.name).includes(event.target.innerText)
+    ) {
       values[index].stage = event.suggestion;
-    } else if (event.target.name && event.target.name.includes('Progress')) {
+    } else if (
+      event.target.innerText &&
+      ['Not Started', 'In Progress', 'Complete'].includes(
```

Comment:
Why are we hard coding these strings? I was expecting that the caller can decide the possible states they want.

Reply:
Looping through the array of suggestions provided to the Progress select:
https://github.com/grommet/grommet-roadmap/blob/master/src/ItemEdit.js#L295

User can still type in anything they want, but selecting an item from the suggestions seems to only include these values.